src/common/cockpitconf.c

```diff
@@ -212,6 +214,38 @@ load_key_file (const char *file_path)
   return ret;
 }
 
+/* Logic to load all additional .conf files from cockpit.conf.d directory */
+static void
+load_conf_d_files (void)
+{
+  const char *conf_d_dir = "/etc/cockpit/cockpit.conf.d";
```

Comment:
We use PACKAGE_SYSCONF_DIR for getting the configuration file path (/etc), so this should also use it as well